apps/site/assets/ts/schedule/__tests__/LineDiagramTest.tsx

```diff
@@ -537,14 +541,60 @@ export const lineDiagram = [
   }
 ] as LineDiagramStop[];
 
+const stops = lineDiagram.map(({ route_stop }) => ({
+  name: route_stop.name,
+  id: route_stop.id,
+  is_closed: false,
+  zone: route_stop.zone || null
+}));
+
 describe("LineDiagram", () => {
-  it("it renders", () => {
-    const wrapper = renderer.create(<LineDiagram lineDiagram={lineDiagram} />);
-    expect(wrapper.toJSON()).toMatchSnapshot();
+  let wrapper: ReactWrapper;
+  beforeEach(() => {
+    wrapper = mount(
+      <LineDiagram
+        lineDiagram={lineDiagram}
+        route={route}
+        directionId={directionId}
+        routePatternsByDirection={routePatternsByDirection}
+        services={[]}
+        stops={{ 0: stops, 1: stops }}
+      />
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  it("renders and matches snapshot", () => {
+    expect(wrapper.debug()).toMatchSnapshot();
```

Comment:
In our current snapshot tests we seem to compare against `renderer.create(...).toJSON()` rather than `wrapper.debug()`. Is there an advantage to this way? (should we start doing it for all our future tests?)

Reply:
Both end up comparing JSON structures so I wouldn't think there'd be an advantage one way or the other; they're just tools that come with different testing libraries. This would be a broader discussion of which testing libraries we use and how we use them and I'd consider this comment outside the scope of a PR review